anchore_engine/services/policy_engine/engine/vulns/mappers.py

```diff
@@ -535,3 +535,99 @@ def to_engine_vulnerabilities(grype_response):
             )
 
     return results
+
+
+class EngineGrypeDBMapper:
+    def _to_engine_vulnerability(self, grype_raw_result):
+        """
+        Receives a single vulnerability_metadata record from grype_db and maps into the data structure engine expects.
+        The vulnerability_metadata record may optionally (but in practice should always) have a nested record for the
+        related vulnerability record.
+        """
+        # Create the templated output object
+        output_vulnerability = {}
+        return_el_template = {
+            "id": None,
+            "namespace": None,
+            "severity": None,
+            "link": None,
+            "affected_packages": None,
+            "description": None,
+            "references": None,
+            "nvd_data": None,
+            "vendor_data": None,
+        }
+        output_vulnerability.update(return_el_template)
+
+        grype_vulnerability = grype_raw_result.GrypeVulnerability
+        grype_vulnerability_metadata = grype_raw_result.GrypeVulnerabilityMetadata
+
+        # Set mapped field values
+        if grype_vulnerability_metadata:
+            output_vulnerability["id"] = grype_vulnerability_metadata.id
+            output_vulnerability[
+                "description"
+            ] = grype_vulnerability_metadata.description
+            output_vulnerability["severity"] = grype_vulnerability_metadata.severity
+            output_vulnerability["link"] = grype_vulnerability_metadata.data_source
+            output_vulnerability[
+                "references"
+            ] = grype_vulnerability_metadata.deserialized_urls
+
+            # vendor_data = {}
```

Comment:
I've temporarily disabled this per @nightfurys . She has another change upcoming that we can pull in here to map this.